spec/system/admin/order_cycles/complex_creating_specific_time_spec.rb

```diff
@@ -31,59 +42,66 @@
     create(:enterprise_relationship, parent: supplier, child: distributor,
                                      permissions_list: [:add_to_order_cycle])
 
-    # And some enterprise fees
-    supplier_fee = create(:enterprise_fee, enterprise: supplier, name: 'Supplier fee')
-    coordinator_fee = create(:enterprise_fee, enterprise: coordinator, name: 'Coord fee')
-    distributor_fee = create(:enterprise_fee, enterprise: distributor, name: 'Distributor fee')
+    shipping_method_i.update!(name: "Pickup - always available")
+    shipping_method_ii.update!(name: "Delivery - sometimes available")
+  end
 
-    # When I go to the new order cycle page
+  it "creating an order cycle with full interface", js: true do
+    ## CREATE
     login_as_admin_and_visit admin_order_cycles_path
     click_link 'New Order Cycle'
 
     # Select a coordinator since there are two available
     select2_select 'My coordinator', from: 'coordinator_id'
     click_button "Continue >"
 
+    fill_in_order_cycle_name
+    select_opening_and_closing_times
+
+    click_button 'Add coordinator fee'
+    select 'Coord fee', from: 'order_cycle_coordinator_fee_0_id'
+
+    click_button 'Create'
+    expect(page).to have_content 'Your order cycle has been created.'
+
+    ## UPDATE
+    add_supplier_with_fees
+    add_distributor_with_fees
+    select_distributor_shipping_methods
+
+    expect_all_data_saved
+  end
+
+  def fill_in_order_cycle_name
     # I cannot save before filling in the required fields
     expect(page).to have_button("Create", disabled: true)
 
     # The Create button is enabled once Name is entered
-    fill_in 'order_cycle_name', with: 'Plums & Avos'
+    fill_in 'order_cycle_name', with: "Plums & Avos"
     expect(page).to have_button("Create", disabled: false)
+  end
 
-    # If I fill in the basic fields
-    find('#order_cycle_orders_open_at').click
-    # select date
-    select_date_from_datepicker Time.zone.at(order_cycle_opening_time)
-    # select time
-    within(".flatpickr-calendar.open .flatpickr-time") do
-      find('.flatpickr-hour').set('%02d' % order_cycle_opening_time.hour)
-      find('.flatpickr-minute').set('%02d' % order_cycle_opening_time.min)
-    end
-    # hide the datetimepicker
-    find("body").send_keys(:escape)
+  def select_opening_and_closing_times
+    select_time("#order_cycle_orders_open_at", order_cycle_opening_time)
+    select_time("#order_cycle_orders_close_at", order_cycle_opening_time)
```

Comment:
This is causing this spec to fail:
https://github.com/openfoodfoundation/openfoodnetwork/actions/runs/3158670190/jobs/5140966992

It should be `order_cycle_closing_time`.

It does not influence the PR, so I'll continue testing :+1: